mlflow/models/container/__init__.py

```diff
@@ -187,9 +187,6 @@ def _serve_pyfunc(model, env_manager):
         procs.append(nginx)
 
     cpu_count = multiprocessing.cpu_count()
-    os.system("pip -V")
-    os.system("python -V")
-    os.system('python -c"from mlflow.version import VERSION as V; print(V)"')
```

Comment:
This change is unrelated. Can you revert?

Reply:
This was from my previous pull request , don't know why it is showing here ?

Reply:
I think it's because you created the `replace-asserttrue` branch from your previous PR's branch.

Reply:
I have deleted old branch, I think it will solve this problem ??

Reply:
It won't because the `replace-asserttrue` branch already contains https://github.com/mlflow/mlflow/pull/6568/commits/85bc04f3c55ef2a9ce5a7766356495b6497a1805.

Reply:
You can reset commits and start from scratch or create a new PR.

Reply:
can i delete this pull request and create a new one ?

Reply:
Yes